physionet-django/physionet/settings/base.py

```diff
@@ -644,3 +644,8 @@ class StorageTypes:
 
 # Emails
 PROJECT_EDITOR_EMAIL = config('PROJECT_EDITOR_EMAIL', default='')
+
+ALLOWED_ACCESS_POLICIES = config(
```

Comment:
We could consider:
- stripping white space from the list
- converting items to all caps